ts/screens/authentication/SpidInformationScreen.tsx

```diff
@@ -37,83 +39,79 @@ const styles = StyleSheet.create({
-      </Row>
-    );
-  }
-
-  private browseToLink() {
-    const url = "https://www.spid.gov.it";
-    // tslint:disable no-floating-promises
-    Linking.openURL(url);
+  private browseToLink(url: string) {
+    Linking.openURL(url).catch(() => {
+      Toast.show({ text: I18n.t("genericError") });
+    });
   }
 
   public render() {
     return (
-      <BaseScreenComponent
-        goBack={true}
-        headerTitle={I18n.t("authentication.spid_information.headerTitle")}
-      >
-        <Content>
-          <H1>{I18n.t("authentication.spid_information.contentTitle")}</H1>
-
-          <View spacer={true} large={true} />
-          <Text>{I18n.t("authentication.spid_information.paragraph1")}</Text>
-          <View spacer={true} extralarge={true} />
-
-          <H1>{I18n.t("authentication.spid_information.subtitle")}</H1>
-          <View spacer={true} />
-          <Text>
-            {I18n.t("authentication.spid_information.paragraph2-part1")}
-            <Text bold={true}>
-              {I18n.t("authentication.spid_information.paragraph2-bold")}
-            </Text>
-            <Text>
-              {` ${I18n.t("authentication.spid_information.paragraph2-part2")}`}
-            </Text>
-          </Text>
-          <Grid>
-            {this.getValueContent(
-              I18n.t("authentication.spid_information.point1-value"),
-              I18n.t("authentication.spid_information.point1-content")
-            )}
-
-            {this.getValueContent(
-              I18n.t("authentication.spid_information.point2-value"),
-              I18n.t("authentication.spid_information.point2-content")
-            )}
-
-            {this.getValueContent(
-              I18n.t("authentication.spid_information.point3-value"),
-              I18n.t("authentication.spid_information.point3-content")
-            )}
-
-            {this.getValueContent(
-              I18n.t("authentication.spid_information.point4-value"),
-              I18n.t("authentication.spid_information.point4-content")
-            )}
-          </Grid>
-          <View spacer={true} extralarge={true} />
-        </Content>
+      <BaseScreenComponent goBack={true}>
+        <Container>
+          <View style={{ margin: 24 }}>
+            <H1>{I18n.t("authentication.spid_information.contentTitle")}</H1>
 
-        <View footer={true}>
-          <Button block={true} primary={true} onPress={this.browseToLink}>
-            <Text>{I18n.t("authentication.spid_information.knowMore")}</Text>
-          </Button>
-        </View>
+            <View spacer={true} large={true} />
+            <Text>{I18n.t("authentication.spid_or_cie")}</Text>
+          </View>
+          <Tabs
+            tabBarUnderlineStyle={styles.tabBarUnderline}
+            tabContainerStyle={styles.tabBarContainer}
+          >
+            <Tab heading={I18n.t("authentication.spid")}>
+              <Content>
+                <Markdown>
+                  {I18n.t("authentication.spid_information.spid")}
```

Comment:
Can you center the number with respect to the item list height? 
Maybe it could be enought to insert `alignItems: 'center'` or `alignSelf: 'center'`
It is:
<img width="167" alt="image" src="https://user-images.githubusercontent.com/38431762/67185655-9b987b80-f3e6-11e9-9570-bc8c7fa638de.png">

and should be:
<img width="325" alt="image" src="https://user-images.githubusercontent.com/38431762/67185668-a521e380-f3e6-11e9-85dc-5758ca99ffbb.png">

If possible, try to increase the number font size.

Reply:
As far as I know, the position of the number of an html list (<li>) cannot be changed, for this reason I used a smaller font. 
It might be possible to do with some other changes to CSS but I can't think of any simple method to do it.

Reply:
To get it the list could be rendered by using again the `getValueContent` function

Reply:
Getting back to using `getValueContent` would "nullify" the changes made by the PR https://github.com/teamdigitale/io-app/pull/1382 whose purpose is to be able to use lists with similar style as it is currently but with the `<Markdown>` component.
In any case, it's okay, I'll do it right now!